src/components/BuildBaronAndAnnotations/AnnotationTicketsTable.tsx

```diff
@@ -1,42 +1,205 @@
+              placement="topRight"
+              title={`Do you want to move this ${issueString} to ${
+                isIssue ? "suspected issues" : "issues"
+              }?`}
+              onConfirm={() => {
+                onClickMove(url, issueKey);
+              }}
+              okText="Yes"
+              cancelText="Cancel"
+            >
+              <StyledButton
+                size="small"
+                data-cy={`move-btn-${issueKey}`}
+                disabled={!userCanModify}
+                glyph={icon}
+              >
+                <StyledText>
+                  Move To {isIssue ? "Suspected Issues" : "Issues"}
+                </StyledText>
+              </StyledButton>
+            </Popconfirm>
+
+            <Popconfirm
+              icon={null}
+              placement="topRight"
+              title={`Do you want to delete this ${issueString}?`}
+              onConfirm={() => {
+                onClickRemove(url, issueKey);
+              }}
+              okText="Yes"
+              cancelText="Cancel"
+            >
+              <StyledButton
+                size="small"
+                data-cy={`${issueKey}-delete-btn`}
+                glyph={<Icon glyph="Trash" />}
+                disabled={!userCanModify}
+              />
+            </Popconfirm>
+          </BtnContainer>
+        </ConditionalWrapper>
+      ),
+    },
+  ];
+  const [removeAnnotation] = useMutation<
+    RemoveAnnotationIssueMutation,
+    RemoveAnnotationIssueMutationVariables
+  >(REMOVE_ANNOTATION, {
+    onCompleted: () => {
+      dispatchBanner.successBanner(
+        `Successfully removed ${isIssue ? "issue" : "suspected issue"}`
+      );
+    },
+    onError(error) {
+      dispatchBanner.errorBanner(
+        `There was an error removing the ${
+          isIssue ? "issue" : "suspected issue"
+        }: ${error.message}`
+      );
+    },
+    refetchQueries: ["GetTask"],
+  });
+
+  const [moveAnnotation] = useMutation<
+    MoveAnnotationIssueMutation,
+    MoveAnnotationIssueMutationVariables
+  >(MOVE_ANNOTATION, {
+    onCompleted: () => {
+      dispatchBanner.successBanner(
+        `Successfully moved ${isIssue ? "issue" : "suspected issue"} to ${
+          isIssue ? "suspected issues" : "issues"
+        }`
+      );
+    },
+    onError(error) {
+      dispatchBanner.errorBanner(
+        `There was an error moving the ${
+          isIssue ? "issue" : "suspected issue"
+        }: ${error.message}`
+      );
+    },
+    refetchQueries: ["GetTask"],
+  });
+
+  const onClickRemove = (url, issueKey) => {
+    const apiIssue = {
+      url,
+      issueKey,
+    };
+    removeAnnotation({ variables: { taskId, execution, apiIssue, isIssue } });
+  };
+
+  const onClickMove = (url, issueKey) => {
+    const apiIssue = {
+      url,
+      issueKey,
+    };
+    moveAnnotation({ variables: { annotationId, apiIssue, isIssue } });
+  };
+
+  return (
+    <TableWrapper>
+      <ErrorBoundary>
```

Comment:
is there a specific error you are meaning to catch with the error boundary?

Reply:
I believe this is part of the first commit, which is not really part of this pr. I branched off of the delete button branch. I explained more in the summary of the PR. Apologies for the confusion!